Cockpit 233 (#356)
cockpit-project/cockpit-project.github.io: _posts/2020-11-25-cockpit-233.md

---
title: Cockpit 233
author: lis
date: 2020-11-25
tags: cockpit linux
slug: cockpit-233
category: release
summary: Cockpit with various bug fixes and performance improvements
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly. Here are the release notes from Cockpit version 233.

> ### Non-admin users no longer see Cockpit in motd
> 
> The motd message suggesting how Cockpit can be enabled or used is now
> shown only to admin users.  This depends on PAM 1.5.0 (which is not yet
> released) but the needed patches have been backported to Fedora 33 and
> may land in other distributions in the future.

Reviewer: ```suggestion
### MOTD is not displayed to non-admin users

The `/etc/motd` message suggesting how Cockpit can be enabled or used is now
shown only to admin users.  This depends on PAM 1.5.0 (which is not yet
released) but the needed patches have been backported to Fedora 33 and
may land in other distributions in the future.

*[MOTD]: Message of the day
```

(I think I did [the abbreviation footnote thing](https://kramdown.gettalong.org/syntax.html#abbreviations) properly?)

Author: thanks for doing this.  i've taken it verbatim.

note that pitti had marked this as release-note, so that's why I included it

Author: I disagree with the motd change, actually.  Some if what you wrote is technically incorrect (ie: this is not `/etc/motd`, nor is it *the* "MOTD")  and I also prefer the headings of release notes to make it clear what the change is rather than stating a simple fact (as your suggested heading does).

> ### Stop building and shipping jQuery
> 
> The long-deprecated `../base1/jquery.js` has been removed from Cockpit.
> All known external users have been ported away.

Reviewer: Does the jQuery removal impact developers? 

If not, I agree with @Gundersanne.

If so, it would be a developer-related note and we should probably show it as such:

```suggestion
## Developers: jQuery removal

Third party developers: Take note! [As previously announced](https://cockpit-project.org/blog/cockpit-163.html), we have finally removed the long-deprcated `../base1/jquery.js` from Cockpit. The bundled version is no longer built or shipped. If you are using jQuery in your application, be sure to ship a version yourself.

All known external users have been ported away.

[Read more details in the original announcement](https://lists.fedorahosted.org/archives/list/cockpit-devel@lists.fedorahosted.org/thread/TDBKI4372I4ZPGQFCZB377DX2QG2YDDY/).
```

---
title: Cockpit 233
author: lis
date: 2020-11-25
tags: cockpit linux
slug: cockpit-233
category: release
summary: Cockpit with various bug fixes and performance improvements
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly. Here are the release notes from Cockpit version 233.

### Non-admin users no longer see Cockpit in motd

The motd message suggesting how Cockpit can be enabled or used is now
shown only to admin users.  This depends on PAM 1.5.0 (which is not yet
released) but the needed patches have been backported to Fedora 33 and
may land in other distributions in the future.

## Developers: jQuery removal

> Third party developers: Take note! [As previously announced](https://cockpit-project.org/blog/cockpit-163.html), we have finally removed the long-deprcated `../base1/jquery.js` from Cockpit. The bundled version is no longer built or shipped. If you are using jQuery in your application, be sure to ship a version yourself.

Reviewer 2: ```suggestion
[As previously announced](https://cockpit-project.org/blog/cockpit-163.html), we have finally removed the long-deprcated `../base1/jquery.js` API from Cockpit. The bundled version is no longer built or shipped. If you are using jQuery in your application, be sure to build against your own copy.
```

> All known external users have been ported away.

Reviewer 2: ```suggestion
All known projects which used this API were ported away two years ago already.
```
